### PR TITLE
update deprecated reference used

### PR DESCRIPTION
**Pull request by aeng**

### CODE REVIEW

File: zanata-client-commands/src/main/java/org/zanata/client/commands/PushPullCommand.java [hunk 1]
```diff
@@ -89,13 +92,8 @@ public PushPullCommand(O opts) {
...
     protected void confirmWithUser(String message) throws IOException {
         if (getOpts().isInteractiveMode()) {
-            Console console = System.console();
-            if (console == null) {
-                throw new RuntimeException(
-                        "console not available: please run Maven from a console, or use batch mode option (-B)");
-            }
-            console.printf(message + "\nAre you sure (y/n)? ");
-            expectYes(console);
+            console.printf(Question, message + "\nAre you sure (y/n)? ");
```
Comment (seanf) [inline zanata-client-commands/src/main/java/org/zanata/client/commands/PushPullCommand.java]: `ConsoleInteractorImpl` doesn't appear to handle the case where `System.console()` is null (when running with `nohup` or something?), so I think this introduces a danger of NPE.

Comment (aeng) [inline zanata-client-commands/src/main/java/org/zanata/client/commands/PushPullCommand.java]: The check should be in ConsoleInteractorImpl() constructor.

**Comment by huangp** [comment]
:+1:

**Comment by seanf** [comment]
:+1:

**Comment by aeng** [comment]
Affected behaviour:

- [ ] Pull statistics and output as CSV
- [ ] Raw file project type push (make sure the console output text properly)
- [ ] Push pull command (other than file project type action, make sure the console output text properly)

**Comment by aeng** [comment]
Merged into integration/master. https://github.com/zanata/zanata-client/commit/cde28583f5648ea65790ad1366c974d625371b24

File: stub-server/src/main/java/org/zanata/rest/service/MockFileResource.java [hunk 2]
```diff
@@ -61,6 +61,7 @@
 @Path(FileResource.SERVICE_PATH)
 public class MockFileResource implements FileResource {
     @Override
+    @Deprecated
```
Comment (seanf) [inline stub-server/src/main/java/org/zanata/rest/service/MockFileResource.java]: This should have an `@deprecated` tag in the javadoc pointing to an undeprecated alternative.

Comment (aeng) [inline stub-server/src/main/java/org/zanata/rest/service/MockFileResource.java]: There's already JavaDoc in the parent class `FileResource`. I'm not sure if we should update all method that implements `FileResource`.

Comment (seanf) [inline stub-server/src/main/java/org/zanata/rest/service/MockFileResource.java]: Well, if `FileResource` has a javadoc with a suitable `@deprecated` tag, that can be inherited (as long as the children don't replace the parent javadoc).

**Pull request closed without merging**